src/components/containerGradient/Display/Display.tsx

```diff
@@ -0,0 +1,48 @@
+import cx from 'classnames';
+
+import styles from './Display.module.scss';
+import DisplayTitle, {
+  Props as DisplayTitleProps,
+} from '../DisplayTitle/DisplayTitle';
+import { ColorLamp, Colors } from '../types';
+
+type Props = {
+  children: React.ReactNode;
+  status?: ColorLamp;
+  color?: ColorLamp;
+  titleProps?: Pick<DisplayTitleProps, 'title' | 'image' | 'children'>;
+  isVertical?: boolean;
+};
+
+function Display({
+  children,
+  titleProps,
+  isVertical,
+  color = Colors.GREEN,
+  status = Colors.GREEN,
+}: Props) {
+  const colorTemp = color || status;
+
+  return (
+    <div
+      className={cx(styles.wrapper, styles[colorTemp], {
+        [styles.vertical]: isVertical,
+      })}
+    >
+      {titleProps && (
```

Comment:
why need to drill titleProps? you can pass DisplayTitle with all the props

Reply:
it is for preventing developers pass everything to header.
By design it is only 1 type of header, so props will limit any extensions

Reply:
That is anti-pattern, it's increase complexity decrease flexibility and make code less understandable.

 if typescript is not enough for you goal, you can check strict type inside "constructor" and throw exception if no Title passed, - usually use such approach.

Reply:
refactored